airflow_provider_hightouch/consts.py

```diff
@@ -0,0 +1,29 @@
+SUCCESS = "success"
+WARNING = "warning"
+CANCELLED = "cancelled"
+FAILED = "failed"
+PENDING = "pending"
+WARNING = "warning"
+SUCCESS = "success"
+QUERYING = "querying"
+PROCESSING = "processing"
+ABORTED = "aborted"
+QUEUED = "queued"
+INTERRUPTED = "interrupted"
+REPORTING = "reporting"
+
+TERMINAL_STATUSES = [CANCELLED, FAILED, SUCCESS, WARNING, INTERRUPTED]
```

Comment:
Not sure about airflow but can you make sure this won't break if we are adding a new status in the future?